client/src/legacy/createEvent.js

```diff
@@ -0,0 +1,20 @@
+/* global window */
+// Polyfill IE11
+function createEvent(type, extraData) {
+  const { document, Event } = window;
```

Comment:
what does this line do?  document and Event are already globally available

Reply:
Makes it very clear where these come from.
It's a small file so could they can be shifted into the global declaration atop the file though I suppose.